### ISSUE TITLE
Last emitted value is not shown

### ISSUE THREAD

**Issue by dzhavat** [swissmanu/rxjs-debugging-for-vscode#56]
## Version Overview

- OS: Windows 10 
- Visual Studio Code Version: 1.56.2 
- "RxJS Debugging for Visual Studio Code" Extension: 0.1.2 
- Node.js: 12.19.1

## Description

The last emitted value is not shown because the text immediately switches to "Unsubscribe". This can also be seen in the demo gif.

In the gif below, the last emitted value should be "Next: 120" but it's not visible.

## How to Reproduce

1. Run the [example code](https://github.com/swissmanu/rxjs-debugging-for-vscode/tree/main/example-workspaces/browser-and-server) as shown in the demo gif

## Expected Behavior

The last emitted value should be shown as well.

## Screenshots/Video

![](https://raw.githubusercontent.com/swissmanu/rxjs-debugging-for-vscode/main/docs/demo.gif)

**Comment by swissmanu** [comment]
Good point @dzhavat! Without checking, I assume the two events are emitted "too close" to each other.

This could be a good use case for #44.

**Comment by dzhavat** [comment]
> Until we have #44 implemented, what do you think about something like "Unsubscribe (Last Value: 102)"?

In order to keep it consistent with the previous messages, I will suggest "Next: 120; Unsubscribe" or "Next: 120 • Unsubscribe" (or something in that direction).
